src/olink/remoteregistry.cpp

```diff
@@ -48,132 +41,100 @@ void RemoteRegistry::removeSource(const std::string& objectId)
 std::weak_ptr<IObjectSource> RemoteRegistry::getSource(const std::string& objectId)
 {
     emitLog(LogLevel::Info, "RemoteRegistry.getObjectSource: " + objectId);
-    std::unique_lock<std::timed_mutex> lock(m_entriesMutex, std::defer_lock);
-    if (lock.try_lock_for(std::chrono::milliseconds(200)))
-    {
-        auto found = m_entries.find(objectId);
-        auto source = found != m_entries.end() ? found->second.source : std::weak_ptr<IObjectSource>();
-        lock.unlock();
-        return source;
-    }
-    else {
-        emitLog(LogLevel::Info, "RemoteRegistry.getObjectSource, registry is busy: " + objectId);
-    }
-    return {};
+    std::unique_lock<std::mutex> lock(m_entriesMutex);
+    auto found = m_entries.find(objectId);
+    auto source = found != m_entries.end() ? found->second.source : std::weak_ptr<IObjectSource>();
+    lock.unlock();
```

Comment:
I think this unlock can be removed since it will be unlocked when leaving the scope?